packages/firebase_auth/firebase_auth_web/lib/src/interop/auth.dart

```diff
@@ -57,7 +34,15 @@ Auth getAuthInstance(App app, {Persistence? persistence}) {
           auth_interop.browserSessionPersistence
         ],
         'popupRedirectResolver': auth_interop.browserPopupRedirectResolver
-      })));
+      }),
+    ),
+  );
+
+  if (persistence != null) {
+    auth.setPersistence(persistence);
```

Comment:
this isn't going to work as we still need to use the `auth_interop.*` value for setting persistence. e.g. `auth_interop.indexedDBLocalPersistence`

Reply:
do you recall that we were able to set `initializeAuth` twice without it erroring? We need to figure out if we can do that instead 🤔

Reply:
I'll circle back to this soon. But as an answer to your first question, `setPersistence` is using `auth_interop.*` in it's implementation if I am not mistaken.